lib/chrono_model/utils.rb

```diff
@@ -51,7 +51,8 @@ def amend_period!(hid, from, to)
 
       connection.execute %[
         UPDATE #{quoted_table_name}
-           SET "validity" = tsrange(#{connection.quote(from)}, #{connection.quote(to)})
+           SET "validity" = tsrange(#{connection.quote(from)}, #{connection.quote(to)}),
+               "recorded_at" = #{connection.quote(from)}
```

Comment:
@algodave yes, makes sense. Feel free to rebase this onto `master` for merging.

Thanks